docs/documentation/platform/pki/subscribers.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "Subscribers"
+sidebarTitle: "Subscribers"
+description: "Learn how to manage PKI subscribers and issue X.509 certificates for them."
+---
+
+## Concept
+
+In Infisical PKI, subscribers are logical representations of entities such as devices, servers, applications that request and receive certificates from Certificate Authorities (CAs).
+
+<div align="center">
+
+```mermaid
+graph TD
+    A[Root CA] --> B[Intermediate CA]
```

Comment:
just to simplify, I think we can omit the root CA -> intermediate CA hierarchy and just have 1 entry with `Certificate Authority`

Reply:
Updated.